Bottom: b30fd88d02d57742c705d70407f4812ad60942e0
Top:    69b300030b2c2d31e4fadeda7fb27a019c31df03
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-08 11:01:06 +0100

Refresh of work-in-progress-on-subcomposi

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b643788..75d12f8 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -868,10 +868,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
      we can't use the hidgl polygon drawing routine */
   /* TODO: We could use the GLU tessellator though */
   if (hidgl_stencil_bits() == 0)
-    {
-      ghid_hid.fill_pcb_polygon = common_fill_pcb_polygon;
-      ghid_hid.poly_dicer = 1;
-    }
+    ghid_hid.fill_pcb_polygon = common_fill_pcb_polygon;
 
   glEnable (GL_BLEND);
   glBlendFunc (GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
